src/main/java/mods/railcraft/common/blocks/machine/worldspike/BlockWorldspikePoint.java

```diff
@@ -84,20 +86,29 @@ public float getBlockHardness(IBlockState state, World worldIn, BlockPos pos) {
 
     @Override
     public float getExplosionResistance(World world, BlockPos pos, Entity exploder, Explosion explosion) {
-//        return 60f * 3f / 5f;
         return 90f;
     }
 
     @Override
     public void defineRecipes() {
         if (RailcraftConfig.canCraftStandardWorldspikes()) {
             CraftingPlugin.addRecipe(getStack(),
-                    " p ",
-                    " o ",
-                    "ogo",
-                    'g', "ingotGold",
-                    'p', Items.ENDER_PEARL,
-                    'o', new ItemStack(Blocks.OBSIDIAN));
+                " p ",
+                " o ",
+                "ogo",
+                'g', "ingotGold",
+                'p', Items.ENDER_PEARL,
+                'o', new ItemStack(Blocks.OBSIDIAN));
         }
     }
+
+    @Override
+    public String getUnlocalizedName() {
+        return super.getUnlocalizedName() + ".name";
```

Comment:
The localization files are supposed to have the ".name" suffix. getUnlocalizedName() returns the tag without the ".name" suffix. getLocalizedName() returns it with it.

Overriding getUnlocalizedName() is definitely the wrong solution for whatever you are trying to do.